README.md

```diff
@@ -187,7 +187,26 @@ cmake .. -G Ninja -DCMAKE_INSTALL_PREFIX=$PWD/../bin -DCMAKE_BUILD_TYPE=Release
 cmake --build . --target install
 ```
 
-### 4) Create desktop shortcut or start AstroMenace
+#### Desktop Entry and Icon Installation
+
+On Linux, the build system can optionally install desktop entry and icon files following the Freedesktop standard. This is disabled by default.
+
+The installation includes:
+- Desktop entry file: `${CMAKE_INSTALL_PREFIX}/share/applications/astromenace.desktop`
+- Icon (64x64): `${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/64x64/apps/astromenace.png`
+- Icon (128x128): `${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/128x128/apps/astromenace.png`

```

Comment:
Is this really work in proper way for FHS systems with current cmake line `INSTALL(TARGETS astromenace DESTINATION ${CMAKE_INSTALL_PREFIX})` for binary files? This looks like, for example, if `CMAKE_INSTALL_PREFIX` set to `/usr` for proper icons and desktop files location, all binary files will be stored in `/usr` itself.